components/channel_header/channel_header.js

```diff
@@ -586,6 +586,16 @@ export default class ChannelHeader extends React.PureComponent {
             pinnedIconClass += ' active';
         }
 
+        let mentionsIconClass = 'channel-header__icon';
+        if (rhsState === RHSStates.MENTION) {
+            mentionsIconClass += ' active';
```

Comment:
Should we use `classnames` pkg for this? It's a bit nicer to manipulate than concatenating strings

Reply:
Looks like @jespino is out, not blocking this pr because of this change. Approving.